src/Resources/js/modules/update-offers-handler.ts

```diff
@@ -1,15 +1,15 @@
 import {ParticipationAction, ParticipationUpdateHandler} from "./participation-update-handler";
 
 export class UpdateOffersHandler {
-    private readonly updateInterval: number = 5000;
+    //private readonly updateInterval: number = 5000;
 
     constructor() {
         this.initCallback();
     }
 
     private initCallback(): void {
         if ($('.button-login').text() !== 'LOGIN') {
-            window.setInterval(this.updateOffers, this.updateInterval);
+           // window.setInterval(this.updateOffers, this.updateInterval);
```

Comment:
What's the purpose of disabling this line?

Reply:
The UpdateOffersHandler has been switched to on demand instead of getting called again after a given intervall, might delete at some point.